grpc/lb/v1/load_balancer.proto

```diff
@@ -45,8 +45,10 @@ message LoadBalanceRequest {
 }
 
 message InitialLoadBalanceRequest {
-  // Name of load balanced service (IE, service.googleapis.com). Its
-  // length should be less than 256 bytes.
+  // The name of the load balanced service (e.g., balancer.service.com). The max
+  // length of the name is 256 bytes.
```

Comment:
256 or 255?

Reply:
I just checked with the LB folks.  We actually don't see apparent limitation to this length in our LB implementation. We definitely need to discuss it and make the requirement clear. Before any decision is made, let me keep the old one.